docs/loop-3/features.md

```diff
@@ -107,12 +107,12 @@ This section is a continuation of the information presented in the [Accept Recom
 * The right side shows a modified bolus greater than the recommended bolus
 * Note that the `Glucose` prediction graph updates based off the `Bolus` value, giving the user the opportunity to accept or change their proposed value before tapping `Save and Deliver`
 * At the next Loop cycle, the app modifies insulin delivery based on the saved information
-    * For the example with bolus less than recommended amount:
+    * For the example with bolus less than the recommended amount:
         * Loop will **NOT** begin to automatically increase insulin delivery until the current glucose is above the bottom of the `Correction` range
         * The recommendation to add insulin when the current glucose is below the `Correction Range` is only offered as a manual feature and is limited to an amount predicted to maintain glucose above the `Safety Threshold`
-    * For the example with bolus greater than recommended amount:
-        * Loop will probably issue an automatic temp basal of 0 U/hr
-        * This is a common "super-bolus" scenario; in other words, "borrow" basal for the meal bolus to limit post meal spikes
+    * For the example with bolus greater than the recommended amount:
+        * Loop will probably issue an automatic temporary basal of 0 U/hr
```

Comment:
To get a correct translation with GT, I need to use `temporary basal' instead of `temp basal'.